### DISCUSSION TITLE
Need help to customize tabline

### DISCUSSION THREAD

**Discussion by GllmR** [nvim-mini/mini.nvim#182] [Q&A] [answered]
Hi, 

i'd like to add an underline on tabs.
I tried to use `MiniTablineCurrent` but i can't figure out how to use it inside `setup` .

Maybe this could be a nice option to have.

**Comment by echasnovski** [comment, marked as answer]
`MiniTablineCurrent` is a [highlight group](https://neovim.io/doc/user/syntax.html#highlight-groups). Use [:highlight](https://neovim.io/doc/user/syntax.html#%3Ahighlight) command to change it.

Try adding this *after* loading your color scheme:
```lua
vim.cmd('hi MiniTablineCurrent gui=underline')
```

**Reply by GllmR** [reply]
Great, thank you !
